index_ja.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">

```

Comment:
`<html lang="ja">`

Reply:
Thank you for pointing it out, and I fixed the lang attribute and some meta attribute too.
I'll make a menu to switch language, so please wait.